amundsen_application/api/mail/v0.py

```diff
@@ -63,6 +74,10 @@ def feedback() -> Response:
             logging.error(message)
 
         return make_response(jsonify({'msg': message}), status_code)
+    except MailClientNotImplemented as e:
```

Comment:
nit: I'm not sure if giving this the same alias `e` will cause any obscure python bugs in the future, but we could always follow the existing convention in our other repos of using `e1`, `e2`, etc